source/jormungandr/jormungandr/street_network/taxi.py

```diff
@@ -133,10 +133,10 @@ def _add_additional_section_after_first_section_taxi(self, journey, pt_object, a
         # Because Jormun does not do it afterwards
         journey.sections[-1].destination.CopyFrom(pt_object)
 
-        self._update_journey(journey, additional_section)
+        self._update_journey(journey, additional_section, True)
```

Comment:
minor: for readability (same on all calls)
```suggestion
        self._update_journey(journey, additional_section, is_add_after=True)
```

Reply:
Thanks, for that! I was looking for some clarity there! :D